ReflectometryServer/parameters.py

```diff
@@ -552,6 +552,45 @@ def is_disabled(self, value: bool):
         self._is_disabled = value
         self.trigger_listeners(ParameterDisabledUpdate(value))
 
+class VirtualParameter(BeamlineParameter):
+    def __init__(self, name: str, axis: ChangeAxis, description: str = None):
+        super(VirtualParameter, self).__init__(name, description=description)
```

Comment:
```suggestion
        super(VirtualParameter, self).__init__(name, description=description, autosave=True)
```

Since this parameter is not linked to a motor axis, the setpoint would be lost on each restart unless we autosave it, so we should always autosave it.